scripts/st2client-startup.sh

```diff
@@ -1,11 +1,16 @@
 #!/bin/bash
 
+counter=0
+backoff=10
+
 # st2client startup and registration
-while true; do
+while [ "$counter" -lt 5 ]; do
   ACTIONS=$(st2 action list)
   if [ "$?" -ne 0 ]; then
-    echo "unable to reach downstream, will try again"
-    sleep 5
+    counter=$((counter+1))
+    echo "unable to reach downstream, will try again in $backoff seconds..."
```

Comment:
Looks like we print the `$backoff` seconds here, while in fact, we sleep `$backoff * 1.5` per next line.
Worth fixing.

Reply:
I see that now. Backoff calc needs to be after sleep.

Reply:
Moved backoff and counter after sleep.